release/rolebased/installer_china.yaml

```diff
@@ -0,0 +1,4110 @@
+apiVersion: v1
```

Comment:
How was this file produced? We have a Make target to produce our installer YAMLs using Kustomize that should be extended (and modified) to also do these.

Reply:
done